docs/docker-stack/entrypoint.rst

```diff
@@ -315,8 +315,8 @@ either as maintenance operations on the database or should be embedded in the cu
 Upgrading Airflow DB
 ....................
 
-If you set :envvar:`_AIRFLOW_DB_UPGRADE` variable to a non-empty value, the entrypoint will run
-the ``airflow db upgrade`` command right after verifying the connection. You can also use this
+If you set :envvar:`_AIRFLOW_DB_MIGRATE` variable to a non-empty value, the entrypoint will run
```

Comment:
Renamed the env var as well for consistency!

Reply:
Good spot!